warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/edge/evaluation/EdgeJexlInterpreter.java

```diff
@@ -39,4 +42,23 @@ public Object visit(ASTOrNode node, Object data) {
         return (matchesL || matchesR);
     }
 
+    @Override
+    protected Object visit(ASTNENode node, Object data) {
+        Object left = node.jjtGetChild(0).jjtAccept(this, data);
+        Object right = node.jjtGetChild(1).jjtAccept(this, data);
+
+        try {
+            if (this.arithmetic instanceof EdgePreconditionArithmetic) {
```

Comment:
It would be nice to avoid instanceof.  Why is !equals not work here?

Reply:
Wanted to leave room for future edge precondition arithmetic implementations